app/forms/name_form.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+class NameForm
+  include ActiveModel::Model
+
+  attr_accessor :trn_request
+  attr_writer :first_name, :previous_first_name, :previous_last_name, :last_name
+
+  validates :first_name, presence: true
+  validates :last_name, presence: true
```

Comment:
Could we add a max length validation of 255 (+ copy the message from another form) to all 4 fields here?

Reply:
@fofr I read your design history post at https://tra-digital-design-history.herokuapp.com/find-a-lost-trn/names/

One suggested example is that the name "Psy", with no first/last name, should actually be a valid "name."

Is that the case in the MVP? Or one to address later if we decide to move to a combined single `Name` field?

Reply:
Kate Lee had some stats on how often this occurred – while there are some records with no first or last names (more commonly first than last), it represents <0.01% in under 65s. So not in MVP.

If we track the validation messages we can see how frequently people attempt to enter only one name.

Reply:
👍